pages/blog/governance-motivation.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Finding a Good Open Governance Model for AsyncAPI"
+date: 2021-03-08T06:00:00+01:00
+type: Strategy
+tags:
+  - governance
+  - foundation
+  - openness
+cover: /img/posts/governance-motivation-cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+  - name: Fran Mendez
+    photo: /img/avatars/fmvilas.webp
+    link: https://twitter.com/fmvilas
+    byline: AsyncAPI Founder
+excerpt: 'AsyncAPI can be successful if the initiative is community-driven. To be community-driven, we need the community to see it can drive things and make an impact. For that, we need a proper open governance model. What model would be the best?'
+featured: true
+---
+
+> tl;dr charter for the AsyncAPI Initiative open governance is ready for the review. Leave your comments there in [this](https://github.com/asyncapi/.github/pull/37) pull request. We will wait two weeks for your feedback.
+
+December 2020 brought to AsyncAPI an important announcement, a [partnership with Postman](/blog/asyncapi-partners-with-postman). This huge milestone secured the AsyncAPI Initiative development efforts as few active community members moved to Postman. This move speeded up our work on transferring AsyncAPI to a foundation and forming an open governance model to assure the community that a single company does not control AsyncAPI Initiative.
+
+It took us a lot of time to write down the initial charter for the initiative.
+
+> **Pro Tip**:
+[Charter](https://dictionary.cambridge.org/dictionary/english/charter) is not a person that creates charts :smiley:
+Nobody's trying to be a smart ass here. We actually had to google that :smiley:
+
+## The vision
+
+We started with a basic vision in our heads and studied governance models of many diverse communities, like NodeJS Foundation, OpenJS Foundation, GraphQL Foundation, Cloud Events, OpenAPI Initiative, and CNCF. We also got a charter template from a foundation. That was a lot to handle. 
+
+<center><img className="w-1/3" src="https://media.giphy.com/media/5QMPpCzH6yxEqhev5A/giphy.gif" /></center>
+
+In the end, we just wanted to make sure the initiative is community-driven and leaves no man behind. We tried to figure out a governance model that:
+- is as democratic as possible
+- supports the asynchronous decision-making process
+- gives power to people that "work", not companies that "pay". In other words, it gives equal power to both individual and corporate contributors.
+
+### Vocabulary
+
+In the governance model, you can find two essential roles, contributor and committer. A contributor is a person that contributes to the project code, docs, or other artifacts. A committer is a person that contributes regularly and is invited by other committers to manage the repository, to have more privileges, and to approve pull requests.
```

Comment:
```suggestion
In this governance model, you can find two essential roles, contributor and committer. A contributor is a person that contributes to the project code, docs, or other artifacts. A committer is a person that contributes regularly and is invited by other committers to manage the repository, to have more privileges, and to approve pull requests.
```

Feel free to ignore this totally :) I might seem very picky, but it sounds way better to me. Like we make it clear that this is just applying to our model.

Reply:
@smoya `this` sounds strange, as there is no discussion about a specific model before, I changed to `the AsyncAPI governance model`, hope this sounds better

Reply:
@derberg Sounds perfect, thanks!